R/system.R

```diff
@@ -268,12 +275,12 @@ queryMongoDB <- function(host = NULL, port = "", database, collection, username,
 #' Returns a data frame that has names of the collections in its "name" column.
 #' @export
 getMongoCollectionNames <- function(host = "", port = "", database = "", username = "",
-                                    password ="", isSSL=FALSE, authSource=NULL, cluster = NULL, additionalParams = "", ...){
+                                    password ="", isSSL=FALSE, authSource=NULL, cluster = NULL, additionalParams = "", timeout = timeout, ...){
```

Comment:
Is it possible to keep additionalParams as the last parameter of this function?

Reply:
updated as suggested.